src/webchat/store/config/config-reducer.ts

```diff
@@ -116,11 +116,11 @@ export const getInitialState = (): ConfigState => ({
 			rating: {
 				enabled: "once",
 				title: "Please rate your chat experience",
-				commentPlaceholder: "Type something here",
+				commentPlaceholder: "Type something here...",
 
 				submitButtonText: "Send feedback",
 				eventBannerText: "Your feedback was submitted",
-				chatStatusMessage: "Feedback submitted",
+				chatStatusMessage: "Feedback submitted", // Not used anywhere. Please see spec and remove if not needed
```

Comment:
I think you can go ahead and remove the chatStatusMessage setting here and in webchat-config.ts